plugins/board/src/index.ts

```diff
@@ -42,9 +42,7 @@ export interface BoardView extends SpaceWithStates {
  * @public
  */
 export interface CardLabel extends AttachedDoc {
-  attachedTo: Ref<Board>
-
-  title: string
+  title?: string
```

Comment:
It really can be undefined?

Reply:
Yes, system labels have only color by default, user defined labels may also have texts.

Reply:
It will be better to use title: string | null, since we do not support undefined for model fields, because mongo doesn't :-)

Reply:
Let's make it not optional and empty if empty/

Reply:
Updated